03_hardware_configuration/slides/slides.md

```diff
@@ -0,0 +1,631 @@
+  <p>Once the certificate has been added, you can find it in the security configuration file and also find the file index. </p>
+  <br>
+  <p>Some commands or blocks break the certificate index. The whole thing then looks like the screenshot shown:</p>
+  <img style="height: 200px; width: 500px" src="./img/SecurityConfiguration-File-Cert.png" />
+
+  <ol>
+   <li><br>For More Informations about the Command check the documentation on the AX Side: <br><a href="https://console.simatic-ax.siemens.io/docs/hw/hwc#import-certificate-command">Import certificate command</a></li> 
+  </ol>
+</div>
+
+----
+
+<header class="slide_header">
+    <h3>User Management UMAC - Define Users and Roles in hwl.yml</h3>
+</header>
+
+<div class="flex-col justify-center">
+  <p>To enable authorization on a PLC, users, passwords, and roles must be assigned. This whole process is referred to as User Management Access Control (UMAC). It consists of two pillars: the assignment of roles and rights, and the assignment of passwords. Passwords are assigned via the CLI and are discussed on the next page. 
+If the config file and PLC Security Configuration File are not kept synchronized, errors may occur during hardware compilation</p>
+  <br>
+  <div class="grid-two-col-eq">
+    <div class="flex-col justify-center">
+    <p>Roles are configured in the hwl.yml file. A <b>"UserManagement"</b> section is created when defining the PLC. There, roles can be defined and then assigned to users.</p>
+    <p>The function roles can come from the following things, for example.:</p>
+    <ul>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/plc-access-protection">The Access to PLC</a></li>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/plc-web-server">PLC Web Server Authorization</a></li>
+     <li><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/opc-ua-server">OPC UA Server Authorization</a></li>
+   </ul>
+    </div>
+    <img style="height: 350px; width: auto" src="./img/UMAC_in_hwl_yml.png" />
+  </div>
+  <ol>
+    <li><br>For More Informations about UMAC on the AX Side: <br><a href="https://console.simatic-ax.siemens.io/docs/hw/security/authorization/user-management#user-management-access-control-umac">User Management Access Control (UMAC)</a></li> 
+  </ol>
+</div>
+
+----
+
+<header class="slide_header">
+    <h3>User Management UMAC - <br>Managing User Credentials over command line of hwc</h3>
+</header>
+
+<div class="flex-col justify-center">
+  <p>Using the command <b>"apax hwc manage-users"</b>, you can list all known users, assign passwords, and create users. As with the other commands, you must specify the PLC name and the location of the PLC config file the user is currently in. Note: Role assignment is still done via the hwl condig file. Therefore, both the config file and user management in the PLC Security Configuration File must be maintained.</p>
```

Comment:
> you must specify the PLC name and the location of the PLC config file 

This can maybe defined more clearly. Hardware Compiler does not need the path to the HWL file that contains the PLC.
Generally, it needs the entire set of HWL files for the project - i.e.  the same `--input` values you would pass to `hwc compile`